metadata-ingestion/src/datahub/ingestion/source/kafka.py

```diff
@@ -252,13 +260,24 @@ def _extract_record(
             topic, topic_detail, extra_topic_config
         )
 
+        # 4. Set dataset's description as top level doc, if topic schema type is avro
+        description = None
+        if (
+            schema_metadata is not None
+            and isinstance(schema_metadata.platformSchema, KafkaSchemaClass)
```

Comment:
check `schema_metadata is not None` is not needed as `isinstance` check is sufficient

Reply:
If we remove **schema_metadata is not None** check, It will throw an exception as nonetype object do not have attribute platformSchema. I guess this check is needed. This check is also present at line no: 249